### DISCUSSION TITLE
域名规则中.baidu.com 与baidu.com 是否相同

### DISCUSSION THREAD

**Discussion by kulongwangzhi85** [pymumu/smartdns#1149] [Q&A]
在domain-list中.baidu.com 与baidu.com 是否相同？
或者在nameserver /baidu.com/cn这样的规则中是不是一样

**Comment by PikuZheng** [comment]
在nameserver和address中，二者一样。在domain-set中，.baidu.com不起作用

**Reply by PikuZheng** [reply]
说起来其实我想知道baidu.com会不会匹配到形如abcbaidu.com。但是一直没找到可靠的域名用来测试

**Reply by kulongwangzhi85** [reply]
感谢

**Reply by zweie** [reply]
也就是说，目前在domain-set中，是只能把域名写全了才能起作用吗？如果说写了baidu.com，是匹配不到pan.baidu.com的咯？

**Reply by PikuZheng** [reply]
> 也就是说，目前在domain-set中，是只能把域名写全了才能起作用吗？如果说写了baidu.com，是匹配不到pan.baidu.com的咯？

不是，不对

**Reply by zweie** [reply]
> > 也就是说，目前在domain-set中，是只能把域名写全了才能起作用吗？如果说写了baidu.com，是匹配不到pan.baidu.com的咯？
> 
> 不是，不对

好的，感谢解答
